src/modality-specific-files/task-events.md

```diff
@@ -98,12 +99,42 @@ In the accompanying JSON sidecar, the `trial_type` column might look as follows:
             "start": "A red square is displayed to indicate starting",
             "stop": "A blue square is displayed to indicate stopping"
         }
+    },
+    "channel": {
+        "Description": "Channel(s) associated with the event",
+        "Delimiter": "|"
+    },
+    "artifact": {
+        "Description": "Description of an artifact."
     }
 }
 ```
 
-Note that all other columns SHOULD also be described but are omitted for the
-sake of brevity.
+Note that in the example above:
+
+1.  Only a subset of columns are described for the sake of brevity.
+    In a real dataset, all other columns SHOULD also be described.
+
+1.  The `channel` column contains a list of values that are separated
+    by a non-default delimiter (that is, *not* with a comma `,`).
+    See the `Delimiter` field in the accompanying JSON sidecar.
+    Thus, the channels related to the event in the third row of the example
+    are called `F,1`, `F,2`, and `Cz`.
+
+1.  The example contains a column called `artifact`.
+    This column is not defined in BIDS, and constitutes additional, arbitrary
+    (that is, unofficial) metadata.
+    In the present case, it is used to describe artifacts in the data,
```

Comment:
Just a suggestion, but I would probably change this to have an HED example, with the clear statement that this is optional, and change to something like: This example describes an artifact in the data, with the optional use of a HED annotation for documenting a Sweat-artifact.

Reply:
done ✅